vscode-extension/package.json

```diff
@@ -86,7 +90,6 @@
             "filenamePattern": "*.aiconfig"
           }
         ],
-        "enableForDiffView": false,
```

Comment:
Why remove this?

Reply:
It was added previously but actually doesn't do anything. The diff view is configured to use default diff view as a result of the code further down:
```
    "configurationDefaults": {
      "workbench.editorAssociations": {
        "{git,gitlens,sapling-diff}:/**/*.{aiconfig.json, aiconfig.yaml}": "default"
      }
    },
```